src/Keyboard.elm

```diff
@@ -42,7 +42,7 @@ keyCode =
 -- MOUSE EVENTS
 
 
-{-| Subscribe to all key presses.
+{-| Subscribe to all presses of keys that normally produce a character value.
```

Comment:
Maybe "Subscribe to all key presses that produce a character value (S, 3, ?, but not Shift, Right Arrow...)"?